src/main/java/ua/danit/final_project/dto/VacancyDto.java

```diff
@@ -0,0 +1,44 @@
+package ua.danit.final_project.dto;
+
+import lombok.Data;
+import ua.danit.final_project.entities.Vacancy;
+
+import java.io.Serializable;
+import java.sql.Timestamp;
+
+
+@Data
+public class VacancyDto implements Serializable {
+
+  private Long id;
+
+  private String position;
+
+  private String salary;
+
+  private String status;
+
+  private String info;
+
+  private Timestamp publication;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused "publication" private field. ([squid:S1068](https://sonarcloud.io/organizations/final-project/rules#rule_key=squid%3AS1068))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=final-project&pullRequest=105&issues=AWXJ1tSGlgbYnohY1fRR&open=AWXJ1tSGlgbYnohY1fRR)

[ISSUE_KEY]: AWXJ1tSGlgbYnohY1fRR